src/components/CurriculumComponents/OverviewTab/OverviewTab.tsx

```diff
@@ -252,9 +253,27 @@ const OverviewTab: FC<OverviewTabProps> = (props: OverviewTabProps) => {
           </OakBox>
         </Box>
       )}
-      <Box $maxWidth={1280} $mh={"auto"} $ph={16} $width={"100%"}>
+      <Box
+        id="curriculum-overview"
+        aria-labelledby="curriculum-overview-heading"
```

Comment:
This is a cool idea, but unfortunately the `aria-labelledby` property is [not allowed](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) on the `generic` role, which is the implicit role for `<div>`. Can the <Box> component have a `tag` of `article` or `section`? Otherwise, you could give it a role of [region](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/region_role) which would then allow the `aria-labelledby` attribute to be used with no further modifications.

Reply:
Good point – I've added the role attribute